pkg/config/ingress_config.go

```diff
@@ -3,10 +3,14 @@ package config
 import "github.com/spf13/pflag"
 
 const (
-	flagIngressClass                      = "ingress-class"
-	flagIngressMaxConcurrentReconciles    = "ingress-max-concurrent-reconciles"
-	defaultIngressClass                   = ""
-	defaultMaxIngressConcurrentReconciles = 3
+	flagIngressClass                         = "ingress-class"
+	flagDisableIngressClassAnnotation        = "disable-ingress-class-annotation"
+	flagDisableIngressGroupNameAnnotation    = "disable-ingress-group-name-annotation"
+	flagIngressMaxConcurrentReconciles       = "ingress-max-concurrent-reconciles"
+	defaultIngressClass                      = "alb"
```

Comment:
yeah, this will be called out in action required section.
I made this change since we encourage user to use IngressClass and deprecate ingress.class annotation via `disable-ingress-class-annotation`. which will require a non-empty ingress.class settings to be useful.